library/hmac_drbg.c

```diff
@@ -387,12 +389,25 @@ void mbedtls_hmac_drbg_set_prediction_resistance( mbedtls_hmac_drbg_context *ctx
     ctx->prediction_resistance = resistance;
 }
 
+/*
+ * Set the reseeding flag
+ */
+void mbedtls_hmac_drbg_set_reseeding( mbedtls_hmac_drbg_context *ctx,
+                                      int reseed_flag )
+{
+    ctx->reseed_flag = reseed_flag;
```

Comment:
If this function takes only two values, it is worth adding an assert. Or change the meaning of it into for example:
`void mbedtls_hmac_drbg_set_reseeding( mbedtls_hmac_drbg_context *ctx, int enable )`. Then for better encapsulation you could move `MBEDTLS_HMAC_DRBG_NO_RESEED` to * .c file and set it inside this function depending on enable